indexer/editable_map_object.cpp

```diff
@@ -161,7 +161,10 @@ void EditableMapObject::SetFlats(string const & flats)
 
 void EditableMapObject::SetBuildingLevels(string const & buildingLevels)
 {
-  m_metadata.Set(feature::Metadata::FMD_BUILDING_LEVELS, buildingLevels);
+  auto constexpr kMaximumLevelsEditableByUsers = 50;
```

Comment:
Как правило, все высокие здания надо мапить отдельно. Илья одобрил

> On Apr 1, 2016, at 14:30, Viktor Govako <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.cpp <https://github.com/mapsme/omim/pull/2670#discussion_r58193408>:
> 
> > @@ -161,7 +161,10 @@ void EditableMapObject::SetFlats(string const & flats)
> >  
> >  void EditableMapObject::SetBuildingLevels(string const & buildingLevels)
> >  {
> > -  m_metadata.Set(feature::Metadata::FMD_BUILDING_LEVELS, buildingLevels);
> > +  auto constexpr kMaximumLevelsEditableByUsers = 50;
> Почему так. Выше была другая константа. Мало ли будут Эмираты мапить ...
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2670/files/0245a28b0babdc0f9da53d96b018371df3fe71df#r58193408>